examples/pa_engine_multiple_unit_example.py

```diff
@@ -49,15 +49,16 @@ def main():
         startdate = "20180101"
         enddate = "20181231"
         frequency = "Monthly"
+        holdings = "B&H"
         # uncomment the below code line to setup cache control; max-stale=0 will be a fresh adhoc run and the max-stale value is in seconds.
         # Results are by default cached for 12 hours; Setting max-stale=300 will fetch a cached result which is 5 minutes older. 
         # cache_control = "max-stale=0"
         get_components_response = components_api.get_pa_components(document=pa_document_name)
         component_id = [id for id in list(
             get_components_response[0].data.keys()) if get_components_response[0].data[id].name == pa_component_name and get_components_response[0].data[id].category == pa_component_category][0]
         print("PA Component Id: " + component_id)
-        pa_accounts = [PAIdentifier(id=portfolio)]
-        pa_benchmarks = [PAIdentifier(id=benchmark)]
+        pa_accounts = [PAIdentifier(id=portfolio, holdingsmode=holdings)]
+        pa_benchmarks = [PAIdentifier(id=benchmark, holdingsmode=holdings)]
```

Comment:
Is it better to include `currency` in this example too?

Reply:
Added support for currency for Multi Unit example

Reply:
@saigiridhar21 can I get approval for this?